packages/dds/tree/src/test/simple-tree/api/getSimpleSchema.spec.ts

```diff
@@ -35,364 +43,609 @@ const simpleNumber: SimpleLeafNodeSchema = {
 };
 
 describe("getSimpleSchema", () => {
-	it("non-copying", () => {
+	useSnapshotDirectory("get-simple-schema");
+
+	describe("non-copying", () => {
```

Comment:
For these tests converted to suites: is the plan to add more tests to them in a future PR?